calico-cloud/get-started/connect/install-cluster.mdx

```diff
@@ -26,6 +26,14 @@ Choose your install option and follow the steps.
 
 1. On the **Managed Clusters** page, click **Connect Cluster**.
 1. Add the **Cluster Name**, and select the **Cluster Type**.
+1. Select the version of {{prodname}} you want to install, we recommend the latest version, which is selected by default
+   :::note
+
+   Currently the only option available to install is the latest version. More versions will become available as newer versions are released
+   and as older versions become qualified.
+
+   :::
+
```

Comment:
I understand the idea here. But I think this note may take away more than it helps. If a user enounters a dropdown with only one version, they're unlikely to get stuck or overly frustrated. 

This time-specific note raises more questions and requires maintenance.

I'd suggest we kill the note entirely.

Reply:
@ozdanborne WDYT?

Reply:
FYI @malavikabala

Reply:
@ctauchen In the absence of @ozdanborne, @tmjd has agreed that this should be removed, which is done now.